### ISSUE TITLE
About the loss of the generator 

### ISSUE THREAD

**Issue by hello-trouble** [junyanz/CycleGAN#152]
Dear professor:
       Thank you for your excellent paper and the project. I am puzzled about the   unsupervised LSGAN loss in my own
project .Both the  loss of G and D are very little, almost near the zero. the figure of the losses are as follows:
![image](https://user-images.githubusercontent.com/33440801/170825986-63aa7517-22c8-4834-a8d7-2f005335c81e.png)
![image](https://user-images.githubusercontent.com/33440801/170826001-ce7a2e7f-ea01-402a-b710-fd17e37a60e7.png)
![image](https://user-images.githubusercontent.com/33440801/170826010-bc0cbcca-f11e-404b-94ea-1a2d67b6f92a.png)
The learning rate are set to 5e-5, Usually, the loss_D tend to be zero with a strong discriminator. However,In my project the loss_G_un is also very lillte, In my opinion, it may be caused by the weiht (0.5 is used) or the training data ? I am now trying a larger weight for the generator.  Would you mind give me some advice ?  Looking forward to your help. Have a good weekend

**Comment by hello-trouble** [comment]
Dear professor:
So sorry to bother you. I have solved this problem  yesterday. Thank you again for your excellent paper and projects.
